From: George McCollister <george.mccollister@gmail.com>
Date: Wed, 17 Nov 2010 10:57:25 -0600
Subject: [PATCH] Switch AC_RUN to AC_LINK

AC_RUN checks fail when cross compiling, switched them to AC_LINK.

Signed-off-by: George McCollister <george.mccollister@gmail.com>
Signed-off-by: Marc Kleine-Budde <mkl@pengutronix.de>
---
 configure.ac |    5 +++--
 1 files changed, 3 insertions(+), 2 deletions(-)

diff --git a/configure.ac b/configure.ac
index 9d2a485..d604de1 100644
--- a/configure.ac
+++ b/configure.ac
@@ -45,11 +45,12 @@ AC_DEFUN([DETECT_SUNCC],
 [
     SUNCC="no"
     AC_MSG_CHECKING([for Sun cc in use])
-    AC_RUN_IFELSE(
+    AC_COMPILE_IFELSE(
       [AC_LANG_PROGRAM([], [dnl
 #ifdef __SUNPRO_C
    return 0;
 #else
+#error not sun cc
    return 1;
 #endif
       ])
@@ -392,7 +393,7 @@ AC_DEFUN([AC_C_HTONLL],
 [
     AC_MSG_CHECKING([for htonll])
     have_htoll="no"
-    AC_RUN_IFELSE([
+    AC_LINK_IFELSE([
        AC_LANG_PROGRAM([
 #include <sys/types.h>
 #include <netinet/in.h>
-- 
1.7.2.3

